sw/host/opentitanlib/src/tpm/driver.rs

```diff
@@ -233,20 +264,34 @@ impl SpiDriver {
         self.spi
             .run_transaction(&mut [spi::Transfer::Both(&req, &mut buffer)])?;
         if buffer[3] & 1 == 0 {
```

Comment:
Maybe leave some comments and/or create consts that explain what is happening with `buffer` here?

Reply:
Done